homeassistant/components/zha/core/channels/general.py

```diff
@@ -355,6 +356,14 @@ def attribute_updated(self, attrid, value):
             async_dispatcher_send(
                 self._zha_device.hass, f"{self.unique_id}_{SIGNAL_ATTR_UPDATED}", value
             )
+            return
+        attr_name = self.cluster.attributes.get(attrid, [attrid])[0]
+        async_dispatcher_send(
+            self._zha_device.hass,
+            f"{self.unique_id}_{SIGNAL_STATE_ATTR}",
+            attr_name,
+            value,
+        )
```

Comment:
Too many arguments are used for this method.

Reply:
Hrm, what do you mean? [`async_dispatcher_send()`](https://github.com/home-assistant/home-assistant/blob/8a78b65f0d99f1383cfdbcea13e95fbe7e3f2288/homeassistant/helpers/dispatcher.py#L77) accepts `*args` and Signal handler is here https://github.com/home-assistant/home-assistant/blob/9cacbdff9fdab4fbea9aa9b13fcb02759c07fd89/homeassistant/components/zha/sensor.py#L211

Reply:
unless I'm missing anything. But would expect it to throw an exception in that case.

Reply:
No sorry you are correct, I was misinterpreting the pylint error.